### PR TITLE
Add correct customer service email

### PR DESCRIPTION
**Pull request by mathilwa**
Based on the locale we need to show different customer service emails

### CODE REVIEW

**Review by hungqt — APPROVED** [review]
🚀

File: src/Nordea/Components/Error.elm [hunk 1]
```diff
@@ -112,12 +115,12 @@ viewActionForInternalServerError config =
             [ Html.text errorActionText
             , FlatLink.default
                 |> FlatLink.view
-                    [ href "mailto: kundeservice.nfe@nordea.com"
+                    [ href (String.append "mailto" customerServiceEmail)
```
Comment (trysnetron) [inline src/Nordea/Components/Error.elm:118]:
```suggestion
                    [ href (String.append "mailto:" customerServiceEmail)
```

Comment (vetlesolgaard) [inline src/Nordea/Components/Error.elm:118]: Do we need the `:` in the link? 😁
```suggestion
                    [ href (String.append "mailto:" customerServiceEmail)
```

Comment (mathilwa) [reply src/Nordea/Components/Error.elm:117]: ah, ofc! it got removed!

**Pull request merged**